### PR TITLE
Add support for new Port SI parameters in PortsOA

### PR DESCRIPTION
**Pull request by tshalvi**
What I did
I added six new SI parameters to the relevant locations in SWSS.

Why I did it
I made these additions to support the new SI parameters recently added to APP_DB.

How I verified it
I verified it using the existing unit tests, and I will be adding new unit tests in the near future.

### CODE REVIEW

File: orchagent/portsorch.cpp [hunk 1]
```diff
@@ -7416,7 +7494,20 @@ bool PortsOrch::setPortSerdesAttribute(sai_object_id_t port_id, sai_object_id_t
     port_serdes_attr.id = SAI_PORT_SERDES_ATTR_PORT_ID;
     port_serdes_attr.value.oid = port_id;
     attr_list.emplace_back(port_serdes_attr);
-    SWSS_LOG_INFO("Creating serdes for port 0x%" PRIx64, port_id);
+    SWSS_LOG_DEBUG("Creating serdes for port 0x%" PRIx64 " with the following SI parameters:", port_id);
```
Comment (prgeor) [inline orchagent/portsorch.cpp:7497]: @tshalvi can we rely upon `/var/log/swss.rec ` for debug log?

Comment (tshalvi) [reply orchagent/portsorch.cpp]: Yea, you're right. I'm removing this log and the following loop.

**Comment by liat-grozovik** [comment]
@prgeor kindly reminder to review this PR.

**Review by prgeor — APPROVED** [review]
@tshalvi LGTM. Will be good to update the PR description where you mentioned adding the logs for debug which is now removed in your latest update

**Comment by prsunny** [comment]
@tshalvi , @prgeor , could you add unit test for this PR?

**Review by keboliu — APPROVED** [review]

**Comment by tshalvi** [comment]
> @tshalvi , @prgeor , could you add unit test for this PR?

According to the checker, it looks like my code changes are 100% covered. Could you please clarify which additional tests are required in this case?

**Comment by prsunny** [comment]
> > @tshalvi , @prgeor , could you add unit test for this PR?
> 
> According to the checker, it looks like my code changes are 100% covered. Could you please clarify which additional tests are required in this case?

there is portsorch unit test which can cover the new sections

**Review by prsunny — APPROVED** [review]

**Pull request merged**